source/runner/DangerUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function sentence(array: Array<string>): string {
```

Comment:
The JSDoc for these functions live in `source/dsl/DangerUtilsDSL.ts`. Is it appropriate to break up the interface and implementation like this?